st2common/st2common/logging/handlers.py

```diff
@@ -37,7 +38,8 @@ def __init__(self, filename, mode='a', maxBytes=0, backupCount=0, encoding=None,
         format_values = {
             'timestamp': timestamp,
             'ts': isotime_str,
-            'pid': pid
+            'pid': pid,
+            'pack': os.path.abspath(os.path.dirname(sys.argv[2])).split('/')[-2]
```

Comment:
There will still only be a single instance of the vSphere sensor running. It will just support multiple vSphere's in its config and spawn threads per vSphere within that process.